### PR TITLE
Updated german language files

### PR DESCRIPTION
**Pull request by TomFreudenberg**

### CODE REVIEW

File: homepage/de.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+  "Integrations": "Integrationen",
```
Comment (nekomajin) [inline homepage/de.js]: How about "**Integration**" (singular), "**Integrieren**", "**Einbinden**" or "**Kombinieren**".

File: homepage/de.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  "Pricing": "Preise",
+  "Apps": "Apps",
+  "Existing user login": "Bestehende Benutzer-Anmeldung",
```
Comment (nekomajin) [inline homepage/de.js]: How about: "Als bestehender Nutzer anmelden" like in [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R5)?

File: homepage/de.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  "login": "Anmelden",
+  "Chat, for GitHub.": "Chat, für GitHub.",
+  "Get unlimited public rooms and one-to-one chats, free forever. Private team plans start from $5 per user per month. No credit card required.": "Unbegrenzt öffentliche Chat-Räume und Direkt-Chats, dauerhaft kostenfrei. Angebote für private Teams beginnen bei $5 USD pro Benutzer und Monat. Es wird keine Kredikarte vorausgesetzt.",
+  "Sign up free with Github": "Kostenfrei anmelden per Github",
```
Comment (nekomajin) [inline homepage/de.js]: Github -> Git**H**ub

File: homepage/de.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  "Sign up with Github": "Anmelden per Github",
```
Comment (nekomajin) [inline homepage/de.js]: Github -> Git**H**ub

File: homepage/de.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  "By signing up you agree to our <a %s>Terms and Conditions</a>": "Durch die Anmmeldung stimmst Du unseren <a %s>allgemeinen Geschäftsbedingungen</a> zu",
+  "Markdown power": "Markdown Unterstützung",
+  "Ever wanted full markdown support in chat? How about syntax highlighting?": "Bislang erfolglos nach einem Chat-System mit markdown Unterstützung gesucht? Wie steht es um Syntax-Highlighting?",
```
Comment (nekomajin) [inline homepage/de.js]: markdown Unterstützung -> Markdown-Unterstützung

File: homepage/de.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  "Yip, we got your back.": "Yep, da haben wir es.",
```
Comment (nekomajin) [inline homepage/de.js]: "Yep, da haben wir **etwas**." :question:

File: homepage/de.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+  "Private conversations": "Private Unterhaltungen",
+  "Public rooms are awesome (and free), but we get that some stuff is private. So we've got <a %s>plans for that</a>.": "Öffentliche Chat-Räume sind ausgezeichnet (und kostenfrei) – Du brauchen aber auch etwas für den internen Austausch? Dafür gibt es <a %s>diese Angebote</a>.",
```
Comment (nekomajin) [inline homepage/de.js]: > Du brauchen aber...

"brauchst"

File: homepage/de.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+  "Developer friendly": "Entwicklerfreundlich",
+  "This is a picture of a cloud against a background of clouds. OMG! We <a %s>integrate</a> with a bunch of stuff in the cloud.": "Dies ist das Bild einer Wolke vor einem Hintergrund aus Wolken. OMG! Wir <a %s>verbinden</a> uns mit vielen Diensten in der Cloud.",
+  "Some of our communities": "Einige aus unseren Gruppen",
+  "Create your own": "Erstell Deine",
```
Comment (nekomajin) [inline homepage/de.js]: "Eigenen Chatroom erstellen" like in [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R20)

File: homepage/de.js [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+  "Explore rooms": "Entdecke Chat-Räume",
+  "Explore": "Entdecken",
+  "Integrated with your workflow": "Verbunden mit Deinem Arbeitsablauf",
+  "Gitter is built on top of GitHub and is tightly integrated with your organisations, repositories, issues and activity.": "Gitter setzt auf GitHub auf und ist so eng mit Deinen Unternehmen und Repositories sowie den Meldungen und Aktivitäten verknüpft.",
+  "We provide integrations with Trello, Jenkins, Travis CI, Heroku, Sentry, BitBucket, HuBoard, Logentries, Pagerduty &amp; Sprintly. We also accept custom webhooks, have an <a %s>open source repository</a> for integrations as well as a <a %s>flexible API</a>.": "Es existieren Schnittstellen zu Trello, Jenkins, Travis CI, Heroku, Sentry, BitBucket, HuBoard, Logentries, Pagerduty &amp; Sprintly. Zusätzlich unterstützen wir individuelle Webhooks und bieten ein <a %s>Open Source Repository</a> für Integrationen sowie eine <a %s>flexible API</a>.",
+  "Loved by our users": "Ist beliebt bei unseren Anwendern",
```
Comment (nekomajin) [inline homepage/de.js]: ~~Ist~~ "**B**eliebt bei unseren Anwendern"

File: homepage/de.js [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+  "Gitter is nothing without the people using it day in and day out.": "Gitter wäre nichts ohne all die Menschen, die es tagein und tagaus nutzen.",
+  "We asked a few what Gitter means to them, this is what they had to say. If you love Gitter, Tweet us <a %s>%s</a> and we'll update these quotes every month.": "Auf unsere Frage 'Was bedeutet Gitter für euch', sagten einige das Folgende. Wenn Dir Gitter auch gefällt, schicke uns einen Tweet an <a %s>%s</a>. Wir aktualisieren die Rückmeldungen jeden Monat.",
```
Comment (nekomajin) [inline homepage/de.js]: > Was bedeutet Gitter für euch,

Remove comma.

File: homepage/de.js [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+  "Personal pricing": "Preise für Einzelpersonen",
+  "Free": "Kostenfrei",
+  "Forever": "Immer",
+  "<b>Unlimited</b> public rooms": "<b>Unbegrentzt</b> öffentliche Chat-Räume",
```
Comment (nekomajin) [inline homepage/de.js]: Typo: Unbegrentzt -> Unbegrenzt

File: homepage/de.js [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+  "<b>1</b> private room": "<b>1</b> privater Chat-Raum",
+  "<b>2 weeks’</b> chat history": "<b>2 Wochen</b> Chat Historie",
+  "<b>Get started</b> for free": "<b>Jetzt kostenfrei starten</b>",
+  "Individual": "Einzelperson",
+  "monthly": "monatlich",
+  "<b>Unlimited</b> private rooms": "<b>Unbegrentzt</b> private Chat-Räume",
+  "<b>Unlimited</b> chat history": "<b>Unbegrentzte</b> Chat Historie",
```
Comment (nekomajin) [inline homepage/de.js]: See [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R34) (without typo :wink:)

File: homepage/de.js [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+  "<b>Purchase</b> now": "Jetzt <b>erwerben</b>",
+  "Organisation pricing": "Preise für Unternehmen",
+  "Bronze": "Bronze",
+  "per user/month": "pro Anwender/Monat",
+  "<b>6 months’</b> chat history": "<b>6 Monate</b> Chat Historie",
```
Comment (nekomajin) [inline homepage/de.js]: [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R44) "Chat-Verlauf"

File: homepage/de.js [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+  "Silver": "Silber",
+  "Our free plan includes one private room per organisation you belong to on GitHub": "Das kostenfreie Angebot erlaubt einen geschützen Chat-Raum pro Unternehmen dem Du auf GitHub zgeordnet bist",
```
Comment (nekomajin) [inline homepage/de.js]: Typo:
- geschützen -> geschützten
- zgeordnet -> zugeordnet

> ...einen geschützen Chat-Raum pro Unternehmen dem Du auf...

"...pro Unternehmen**,** dem Du ..."" -> add comma

File: homepage/de.js [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+  "Our Individual plan allows you to create private rooms using your personal account as the owner": "Das Angebot für Einzelpersonen erlaubt es, geschützte Chat-Räume im Namen Deines persönlichen Benutzerkontos zu erstellen",
+  "Our organisation plans allow you to create private rooms using your GitHub organisation as the owner": "Das Angebot für Unternehmen erlaubt es, geschützte Chat-Räume im Namen Deines GitHub Unternehmens zu erstellen",
+  "Prices may be subject to VAT. By signing up you agree to our <a %s>Terms and Conditions</a>.": "Angegebene Preise unterliegen ggf. noch der gesetztlichen Mehrwertsteuer. Durch Deine Anmeldung stimmst Du unseren <a %s>allgemeinen Geschäftsbedingungen</a> zu.",
+  "Get Gitter now!": "Nutze Gitter jetzt!",
+  "Gitter is available in all modern browsers as well as apps for desktops and mobile phones.": "Gitter ist verfügbar für alle modernen Browsern und den zusätzlichen Apps für Desktop und Smartphone.",
```
Comment (nekomajin) [inline homepage/de.js]: Typo: Browsern -> Browser

"Gitter ist verfügbar für alle modernen Browser und zusätzlich als Apps für Desktop und Smartphone."

File: homepage/de.js [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+  "We currently have apps for OSX, iOS, Android, Windows and Linux. We also have a sweet little <a %s>IRC bridge</a>": "Zur Zeit verfügen wir über Apps für OSX, iOS, Android, Windows und Linux. Zusätzlich gibt es auch eine kleine schlanke <a %s>IRC Bridge</a>",
+  "Mac": "Mac",
+  "Windows": "Windows",
+  "Linux 32": "Linux 32",
+  "Linux 64": "Linux 64",
+  "iPhone": "iPhone",
+  "Android": "Android",
+  "By developers, for developers": "Von Entwicklern, für Entwickler",
+  "Gitter is built in London by a team of developers just like you. Hell even the product guy writes code, although we generally end up rewriting those bits.": "Gitter wurde von einer Gruppe von Entwicklern wie Dir in London gebaut. Selbst der Produkt-Mensch schreibt Code, obwohl das doch immer damit endet, dass wir diese Bits nochmal umschreiben.",
```
Comment (nekomajin) [inline homepage/de.js]: [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R59)

"Gitter wird von einer Gruppe von Entwicklern wie Dir in London gebaut. Selbst der Produktmanager schreibt Code, obwohl das immer damit endet, dass wir diese Bereiche nochmal umschreiben."

File: homepage/de.js [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+  "If you're ever around Silicon Roundabout, give us a shout, we love meeting up with other developers.": "Falls Du mal in der Gegend um Silicon Roundabout bist, freuen wir uns über Deinen Kontakt – wir lieben den Austausch mit anderen Entwicklern.",
+  "Support": "Support",
```
Comment (nekomajin) [inline homepage/de.js]: Maybe "**Hilfe**" or "**Unterstützung**"?

File: homepage/de.js [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+  "API": "API",
+  "Blog": "Blog",
+  "Twitter": "Twitter",
+  "Gitter &mdash; Where developers come to talk.": "Gitter &mdash; Where developers come to talk.",
```
Comment (nekomajin) [inline homepage/de.js]: [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R65) -> "Gitter &mdash**;** Wo sich Entwickler austauschen."

**Comment by nekomajin** [comment]
@TomFreudenberg The translations sound good. I have only small improvements.

**Comment by TomFreudenberg** [comment]
Hi Martin ( @nekomajin )

sorry for late feeedback during new years time. Btw. Happy new year ;-)

I have checked your improvements and can agree, should I update the sources and PR?

How do you like to proceed?

**Comment by MadLittleMods** [comment]
@TomFreudenberg Keep it in the same PR. Update the source and just force push to the same branch and it will auto-resolve the comments once each piece is touched.

**Comment by nekomajin** [comment]
Hey @TomFreudenberg. No problem. I wish you also a happy new year :smile: :fireworks:
I agree with @MadLittleMods: `push` your changes to the same PR and we all will be happy :tada:

**Comment by MadLittleMods** [comment]
@TomFreudenberg Just checking in :)

**Comment by TomFreudenberg** [comment]
Hey Eric ( @MadLittleMods )

sorry just not on focus .... I will prepare PR on Weekend

Thanks for ping
Tom

**Comment by TomFreudenberg** [comment]
Hi Eric, Martin ( @MadLittleMods @nekomajin )

I pushed my updates and hope everything is fine by now. See my comments and thanks for your feedback.

Tom

**Comment by MadLittleMods** [comment]
I don't speak German so I would love a second opinion after the changes from review. Thank you @TomFreudenberg for touching this again!

**Comment by nekomajin** [comment]
@TomFreudenberg @MadLittleMods Just from one typo, I find the changes top :thumbsup:

**Comment by TomFreudenberg** [comment]
Hey Martin ( @nekomajin )

thanks for review. I already had changed your marked typo at commit https://github.com/TomFreudenberg/gitter-translations/commit/5468b18947ad11eefd0e867e9368e9dfa23b7c44

So if you take a look at change de.js file, all is in place.

So hopefully some one will use it then.

Cheers
Tom

**Comment by MadLittleMods** [comment]
It's not clear to me why that comment wasn't dismissed as it does seem to be addressed here. Merging now.

**Comment by TomFreudenberg** [comment]
:+1: thanks for merging

**Pull request merged**